contracts/src/PDPService.sol

```diff
@@ -55,7 +55,8 @@ contract PDPService {
     mapping(uint256 => uint256) nextRootId;
     mapping(uint256 => uint256) proofSetLeafCount;
     mapping(uint256 => uint256) nextChallengeEpoch;
-    mapping(uint256 => address) proofSetRecordKeeper;
+    // Each proof set notifies a configurable application managing data storage
+    mapping(uint256 => address) proofSetApplication;
```

Comment:
I would just call this a receiver or a listener. This contract doesn't care what it is or how it works. This naming approach then applies to many changes below, too.

Reply:
listener is way better